ansible.cfg

```diff
@@ -9,3 +9,5 @@ retry_files_save_path = $HOME/.ansible/retry-files
 remote_user = vagrant
 host_key_checking = False
 force_color = 1
+roles_path= ./roles
```

Comment:
On line 7 we already have a `roles_path`. Doesn't that work for you? It'd be good to have just 1 definition.

Reply:
@ekohl if you see the error in the comment, ansible is not able to resolve $PWD variable in Ubuntu, although this works completely fine in fedora. Waiting for the ansible folks reply on this one.

the path being resolved is like : `/home/rabajaj/forklift/$PWD/roles`, trying to figure out why $PWD is being intact here in the path.